src/utils/mail/sending-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { google, gmail_v1 } from 'googleapis';
+import { OAuth2Client } from 'google-auth-library';
+import { GaxiosResponse } from 'gaxios';
+
+/**
+ * TODO: Creat an email message and encode it to Base64
+ */
+export const createEmailMessage = (requestBody: string): string => { // eslint-disable-line
+    // A hardcode test email message
+    // Note: Please use a Base64 text encoder/decoder to customize your email message
+    // and remember to change the recipient for testing purpose
+    return 'RnJvbTogPEZST01AZ21haWwuY29tPgpUbzogPG5hbmN5d2FuMTAwNEBnbWFpbC5jb20+ClN1YmplY3Q6IFRlc3QgRW1haWwKClRlc3Q=';
+};
```

Comment:
I would change this to be `encodeEmail` and take some relevant strings.

Converting to Base64 is pretty simple in JS so you can probably make this function fully featured for this PR - https://stackoverflow.com/questions/246801/how-can-you-encode-a-string-to-base64-in-javascript